controller/controller.go

```diff
@@ -289,6 +289,12 @@ func (f *Controller) bootWithError(ctx context.Context) error {
 		// TODO collect metrics
 	}
 
+	f.logger.LogCtx(ctx, "function", "bootWithError", "level", "debug", "message", "booting informer")
```

Comment:
I would like to have logs for the end of the event as well. 

As another note, I am not sure if this function key is really useful. Anybody anything against dropping it?

Reply:
Nothing against it.

Reply:
I added logs at the end, let's remove the `function` key in another PR as it used in a few places.